### DISCUSSION TITLE
New to WFDS

### DISCUSSION THREAD

**Discussion by ahmedahmed2056** [firemodels/fds#12740] [General]
Hello. I am new to WFDS and I am trying to run the mell 2009 douglas fir tree file. First is WFDS needs to be downloaded and installed separatel from fds? or it is integrated with the fds i already installed ?

second : when I run the tree_2_m_14_pc.fds with fds_local, it gives me an error saying (ERROR: Problem with MATL number 4, line number 60 (CHID: tree_2_m_14_pc)) , what is the problem here especially this is the file uploaded by ruddy Mell so it should be working fine

here is the code 
&HEAD CHID='tree_2_m_14_pc', TITLE='2 m Douglas Fir with 14% moisture content' /

&MESH IJK=40,40,80, XB=-1.0,1.0,-1.0,1.0,0.0,4.0 /

&TIME T_END=30. / 

&REAC FUEL='CELLULOSE', C=2.28, H=6.34, O=2.44, SOOT_YIELD=0.02, HEAT_OF_COMBUSTION=14516. /

&SPEC ID='WATER VAPOR' /

&VENT MB='XMIN', SURF_ID='OPEN' /
&VENT MB='XMAX', SURF_ID='OPEN' /
&VENT MB='YMIN', SURF_ID='OPEN' /
&VENT MB='YMAX', SURF_ID='OPEN' /
&VENT MB='ZMAX', SURF_ID='OPEN' /
&VENT MB='ZMIN', SURF_ID='FOIL' /

&SURF ID='FOIL'
      COLOR = 'SILVER'
      MATL_ID='ALUMINUM','CONCRETE'
      THICKNESS(1:2) = 0.0002,0.01 /

&MATL ID='ALUMINUM'
      DENSITY = 2707.
      SPECIFIC_HEAT = 0.896
      CONDUCTIVITY = 204.
      EMISSIVITY = 0.1 /

&MATL ID='CONCRETE'
      DENSITY = 1500.
      SPECIFIC_HEAT = 1.
      CONDUCTIVITY = 0.3  /

&SURF ID                        = 'foliage'
      MATL_ID                   = 'vegetation'
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 3940.
      LENGTH                    = 0.05
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'small roundwood'
      MATL_ID                   = 'vegetation'
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 2667.
      LENGTH                    = 0.10
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'medium roundwood'
      MATL_ID                   = 'vegetation'
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 888.
      LENGTH                    = 0.10
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'large roundwood'
      MATL_ID                   = 'vegetation'
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 500.
      LENGTH                    = 0.10
      GEOMETRY                  = 'CYLINDRICAL' /

&MATL ID = 'vegetation'
      DENSITY = 514.
      CONDUCTIVITY = 0.11
      SPECIFIC_HEAT_RAMP = 'cp_dry_veg'
      A = 36300.
      E = 60300.
      NU_SPEC = 0.74
      SPEC_ID = 'CELLULOSE'
      NU_MATL = 0.26
      MATL_ID = 'CHAR'
      HEAT_OF_REACTION = 416. /

&RAMP ID='cp_dry_veg', T= 0.,    F=1.18 /
&RAMP ID='cp_dry_veg', T= 1000., F=4.81 /

&MATL ID                 = 'CHAR'
      DENSITY            = 300.
      CONDUCTIVITY       = 0.052
      SPECIFIC_HEAT_RAMP = 'cp_char'
      N_S                = 0.
      NU_O2_CHAR         = 1.65
      BETA_CHAR          = 0.2
      A                  = 430.
      E                  = 74800.
      SPEC_ID            = 'PRODUCTS','AIR'
      NU_SPEC            = 8.18,-7.2
      MATL_ID            = 'ASH'
      NU_MATL            = 0.02
      HEAT_OF_REACTION   = -12000. /

&RAMP ID='cp_char', T=   27, F=0.715 /
&RAMP ID='cp_char', T= 1724, F=2.04 /

&MATL ID            = 'ASH'
      DENSITY       = 67.
      CONDUCTIVITY  = 0.1
      SPECIFIC_HEAT = 2.0 /

&PART ID='foliage', DRAG_COEFFICIENT=2.8, SHAPE_FACTOR=0.25, SAMPLING_FACTOR=10, SURF_ID='foliage'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS', STATIC=.TRUE., COLOR='FOREST GREEN' /
&INIT PART_ID='foliage', XYZ=0.0,0.0,0.15, RADIUS=0.85, HEIGHT=1.9, SHAPE='CONE', N_PARTICLES_PER_CELL=1, PACKING_RATIO=0.0032 /

&PART ID='small roundwood', DRAG_COEFFICIENT=2.8, SHAPE_FACTOR=0.25, SAMPLING_FACTOR=10, SURF_ID='small roundwood'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS','PARTICLE DIAMETER', STATIC=.TRUE., COLOR='BROWN' /
&INIT PART_ID='small roundwood', XYZ=0.0,0.0,0.15, RADIUS=0.85, HEIGHT=1.9, SHAPE='CONE', N_PARTICLES_PER_CELL=1, PACKING_RATIO=0.00055 /

&PART ID='medium roundwood', DRAG_COEFFICIENT=2.8, SHAPE_FACTOR=0.25, SAMPLING_FACTOR=10, SURF_ID='medium roundwood'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS','PARTICLE DIAMETER', STATIC=.TRUE., COLOR='BROWN' /
&INIT PART_ID='medium roundwood', XYZ=0.0,0.0,0.15, RADIUS=0.85, HEIGHT=1.9, SHAPE='CONE', N_PARTICLES_PER_CELL=1, PACKING_RATIO=0.0005 /

&PART ID='large roundwood', DRAG_COEFFICIENT=2.8, SHAPE_FACTOR=0.25, SAMPLING_FACTOR=10, SURF_ID='large roundwood'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS','PARTICLE DIAMETER', STATIC=.TRUE., COLOR='BROWN' /
&INIT PART_ID='large roundwood', XYZ=0.0,0.0,0.15, RADIUS=0.85, HEIGHT=1.9, SHAPE='CONE', N_PARTICLES_PER_CELL=1, PACKING_RATIO=0.00075 /

&PART ID='ignitor', SAMPLING_FACTOR=1, SURF_ID='burner', STATIC=.TRUE. /
&SURF ID='burner', HRRPUA=180., RAMP_Q='ignite', RADIUS=0.01, GEOMETRY='SPHERICAL' /
&RAMP ID='ignite', T= 0.0, F=0. /
&RAMP ID='ignite', T= 0.1, F=1. /
&RAMP ID='ignite', T=10.0, F=1. /
&RAMP ID='ignite', T=10.1, F=0. /
&INIT PART_ID='ignitor', XYZ=0.0,0.0,0.21, RADIUS=0.40, SHAPE='RING', N_PARTICLES=400  /

&DEVC XB=-1.0,1.0,-1.0,1.0,0.0,4.0, QUANTITY='MPUV', PART_ID='foliage', ID='foliage', STATISTICS='VOLUME INTEGRAL'  /
&DEVC XB=-1.0,1.0,-1.0,1.0,0.0,4.0, QUANTITY='MPUV', PART_ID='small roundwood', ID='small roundwood', STATISTICS='VOLUME INTEGRAL'  /
&DEVC XB=-1.0,1.0,-1.0,1.0,0.0,4.0, QUANTITY='MPUV', PART_ID='medium roundwood', ID='medium roundwood', STATISTICS='VOLUME INTEGRAL'  /
&DEVC XB=-1.0,1.0,-1.0,1.0,0.0,4.0, QUANTITY='MPUV', PART_ID='large roundwood', ID='large roundwood', STATISTICS='VOLUME INTEGRAL'  /

&SLCF PBY=0., QUANTITY='RADIATION LOSS', CELL_CENTERED=.TRUE. /
&SLCF PBY=0., QUANTITY='INTEGRATED INTENSITY', CELL_CENTERED=.TRUE. /
&SLCF PBY=0., QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF PBY=0., QUANTITY='TEMPERATURE', VECTOR=.TRUE. /

&BNDF QUANTITY='WALL TEMPERATURE' /
&BNDF QUANTITY='GAUGE HEAT FLUX' /

&TAIL /

**Comment by drjfloyd** [comment]
WFDS features are now in the current release; however, some FDS inputs have changed in the decade plus since WFDS was bracnhed off.  I think there is a version of this case in the current FDS repository. You can look for it there.

**Reply by ahmedahmed2056** [reply]
Thank you .. I already find a more recent file in the FDS repository rather than the one in WFDS repository which is older file. So now when i run, i run with fds_local in the cmd ? or there is another command for WFDS ? 

THANKS ALOT

**Reply by drjfloyd** [reply]
just run FDS as normal.
